src/components/Hero/Hero.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Hero.css';
+
+const propTypes = {
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+};
+
+function Hero(props) {
+  return (
+    <main className="demo-moveout">
+      <section className="content content--layout">
+        <h2 className="word word--moveout">{props.title}</h2>
+        <h4 className="f3 content-center: center">{props.description}</h4>
+        <div className="flex items-center justify-center pa4 mt6 bg-alert-blue text-our-gray">
+          <svg className="w1" data-icon="info" viewBox="0 0 32 32">
+            <title>info icon</title>
+            <path d="M16 0 A16 16 0 0 1 16 32 A16 16 0 0 1 16 0 M19 15 L13 15 L13 26 L19 26 z M16 6 A3 3 0 0 0 16 12 A3 3 0 0 0 16 6" />
+          </svg>
+          <span className="lh-title ml3">We&apos;re building some exciting tools in this space; stay tuned. <br /> Meanwhile, play around with existing datasets below or <a href="mailto:team@recidiviz.com">contact us.</a></span>
```

Comment:
Does the apostrophe encoding need to be made explicit here?

Reply:
Also, let's inject `contactEmail` here.

Reply:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md

This is a rule we could choose to warn on (or ignore) instead but I think it's a decent one for the reasons outlined above.

Reply:
Oh it turns out I had the exact wrong opinion on this. I change my mind!